lib/backend_common/backend_common/notifications.py

```diff
@@ -17,3 +26,69 @@
 URGENCY_LEVELS = [
     'LOW', 'NORMAL', 'HIGH',
 ]
+
+
+def get_current_app_credentials() -> dict:
+    return {
+        'id': current_app.config.get('TASKCLUSTER_CLIENT_ID'),
```

Comment:
access this key directly since we are making sure in ``settings.py`` that it exists.

```
'id': current_app.config['TASKCLUSTER_CLIENT_ID'],
```